src/main/resources/templates/template_hmpps-offender-categorisation-api.mustache

```diff
@@ -26,7 +26,7 @@
             <tr>
                 <td>{{ optionalValue (formatDate referred_date) }}</td>
                 <td>{{ optionalValue status }}</td>
-                <td>{{ optionalValue prison_id }}</td>
+                <td>{{ getPrisonName prison_id }}</td>
```

Comment:
Just wanted to check that we don't need to do this for approved_placement_prison_id or placement_prison_id further down in this template too?

Reply:
yeah saw that - I need to check with the cat team what they return to us as the example didn't have data in